source/guide_grafana-loki.rst

```diff
@@ -0,0 +1,277 @@
+.. highlight:: console
+
+.. author:: Franz Wimmer <https://codefoundry.de>
+
+.. tag:: lang-go
+.. tag:: monitoring
+.. tag:: logs
+.. tag:: logshipper
+.. tag:: observability
+.. tag:: audience-admins
+
+.. sidebar:: About
+
+  .. image:: _static/images/loki.png
+      :align: center
+
+############
+Grafana Loki
+############
+
+.. tag_list::
+
+Loki is a log aggregation system designed to store and query logs from all your applications and infrastructure.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`supervisord <daemons-supervisord>`
+  * :lab:`Grafana <guide_grafana>`
+
+License
+=======
+
+Loki is licensed under the GNU Affero General Public License (AGPL).
+
+All relevant legal information can be found here
+
+  * https://github.com/grafana/loki/blob/main/LICENSE
+
+Prerequisites
+=============
+
+We need to prepare a couple of directories and create a database which is used by grafana.
+
+Directory for building loki from source:
+
+::
+
+ [isabell@stardust ~]$ mkdir -p ~/software
+ [isabell@stardust ~]$
+
+Directory for storing the custom configuration files:
+
+::
+
+ [isabell@stardust ~]$ mkdir -p ~/etc/loki
+ [isabell@stardust ~]$
+
+Directory for storing the log files:
+
+::
+
+ [isabell@stardust ~]$ mkdir -p ~/var/log/grafana
+ [isabell@stardust ~]$
+
+Loki Installation
+=================
+
+Clone loki from the official git repository:
+
+::
+
+ [isabell@stardust ~]$ git clone https://github.com/grafana/loki ~/software/
+ [isabell@stardust ~]$ cd ~/software/loki
+ [isabell@stardust loki]$
+
+Build loki
+----------
+
+::
+
+ [isabell@stardust loki]$ make loki
+
+::
+
+Move the binary to ``~/bin`` and the default configuration file to ``~/etc/loki``.
+
+::
+
+ [isabell@stardust loki-2.6.1]$ mv cmd/loki/loki ~/bin/
+ [isabell@stardust loki-2.6.1]$ mv cmd/loki/loki-local-config.yaml ~/etc/loki/loki.yaml
+
+Loki Configuration
+------------------
+
+
+Create the file ``~/etc/loki/loki.yaml`` with the following content:
```

Comment:
we just moved this file there, should we just edit some lines or rather remove it and use this content?